src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -281,6 +281,52 @@ Below is a list of available validate commands:
 | `--kubeconfig` `<string>`    | Absolute path(s) to the kubeconfig file(s) (default `$HOME/.kube/config`)
 | `--kubecontext` `<string>`   | Kubeconfig context to use
 
+### `gather`
+
+The `subctl gather` command is a tool that collects various information from clusters to aid in troubleshooting a
+Submariner deployment, including Kubernetes resources and Pod logs. Clusters from which information is gathered are provided
+through `--kubeconfig` flag (or `KUBECONFIG` env var). By default it will gather information from all clusters contexts provided
+in kubeconfig. To gather information from specific clusters, contexts can be passed using `kubecontexts` flag.
+
+The tool creates a UTC timestamped directory of the format `submariner-YYYYMMDDHHMMSS` containing various files.
+Kubernetes resources are written to YAML files with the naming format `<cluster-name>_<resource-type>_<namespace>_<resource-name>.yaml`.
+Pod logs are written to files naming format `<cluster-name>_<pod-name>.log`
+
+The specific information collected is configurable. As part of `resources` for `connectivity` module it also gathers troubleshooting
+data specific for the CNI and CableDriver from respective nodes using file format `<cluster-name>_<node-name>_<command>.yaml`
+
+#### `gather` flags
+
+| Flag                       | Description
+|:---------------------------|:--------------------------------------------------------------------------------------------------------------------------|
+| `--kubeconfig` `<string>`  | Absolute path(s) to the kubeconfig file(s). By default, it iterates over all kubeconfigs
+| `--kubecontexts` `<string>`| comma separated list of kube contexts to use. By default all contexts referenced by kubeconfig are used
+| `--module` `<string>`      | Comma-separated list of components for which to gather data. Default is `operator,connectivity,service-discovery,broker`
+| `--type` `<string>`        | Comma-separated list of data types to gather. Default is `logs,resources`
+
+#### `gather` examples
+
+These examples assume that kubeconfigs have been passed using `KUBECONFIG` env var. Alternately, add `--kubeconfig` flag to these if
+env var is not set.
+
+##### `gather` all from all clusters
+
+Users are recommended to use this when reporting any issue.
```

Comment:
```suggestion
It is recommended to use this when reporting any issue.
```

Reply:
@dfarrell07 @mangelajo @tpantelis - I think we should also edit our GitHub Bug Report template and instruct the reporter to use `subctl gather` there, does it make sense?